spec/scrabble_spec.js

```diff
@@ -205,7 +205,24 @@ describe('Player', function() {
       player.play('zzzz');
       expect(player.highestScoringWord()).toBe('zzzz');
     });
-
+    // It must also work for a longer list of words.
+    it('returns the highest scoring word played', function() {
+      let player = new Scrabble.Player('test player');
+      player.play('cat');
```

Comment:
I think i'd want their tests to use an array of words, iterate and play each word and then check the highest to make this less repetitive